components/telemetry-operator/controllers/logpipeline_controller.go

```diff
@@ -311,18 +306,27 @@ func (r *LoggingConfigurationReconciler) deleteFluentBitPods(ctx context.Context
 	return nil
 }
 
-// Merge list of Sections to single FluentBit configuration.
-func generateFluentBitConfig(sections []telemetryv1alpha1.Section) string {
+// Merge FluentBit parsers, filters and outputs to single FluentBit configuration.
+func mergeFluentBitConfig(parsers []telemetryv1alpha1.Parser, multiLineParsers []telemetryv1alpha1.MultiLineParser, filters []telemetryv1alpha1.Filter, outputs []telemetryv1alpha1.Output) string {
 	var result string
-	for _, section := range sections {
-		result += section.Content + "\n\n"
+	for _, parser := range parsers {
+		result += "[PARSER]\n\t" + parser.Content + "\n\n"
```

Comment:
As we discussed, the tab char is most probably not needed here. Could you please check?